docs/index.md

```diff
@@ -1087,16 +1087,30 @@ Files specified using `--file` _are not affected_ by this option.
 
 Can be specified multiple times.
 
-### `--extension <ext>, --watch-extensions <ext>`
-
-> _Updated in v6.0.0. Previously `--watch-extensions`, but now expanded to affect general test file loading behavior. `--watch-extensions` is now an alias_
+### `--extension <ext>`
 
 Files having this extension will be considered test files. Defaults to `js`.
```

Comment:
> --global can accept a comma-delimited list; --global app,YUI is equivalent to --global app --global YUI.

Please add the equivalent information for `--extension`.

Reply:
https://github.com/mochajs/mocha/pull/3980/commits/b422fa8f8ecc2d01beecc62e72f7463c2dba8075